Formula/harfbuzz.rb

```diff
@@ -12,46 +13,39 @@ class Harfbuzz < Formula
     sha256 "9ca4eaac80b2db11439e865f977d6b7b330b2bd5ca8f558e23be49acda67b23e" => :high_sierra
   end
 
-  head do
-    url "https://github.com/behdad/harfbuzz.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-    depends_on "ragel" => :build
-  end
-
   depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "ragel" => :build
```

Comment:
`ragel` shouldn’t be required, the generated files are committed to the repository.

Reply:
@khaledhosny thanks, you are correct. It looks like `pkg-config` and `gobject-introspection` are also not required to be `:build` dependencies, so the ` depends_on` part should just be:
```
  depends_on "meson" => :build
  depends_on "ninja" => :build
  depends_on "cairo"
  depends_on "freetype"
  depends_on "glib"
  depends_on "gobject-introspection"
  depends_on "graphite2"
  depends_on "icu4c"
```

Reply:
Actually, since `gobject-introspection` has `cairo` and `glib` as dependencies, and `cairo` has `freetype` as a dependency, perhaps `harfbuzz` just requires:
```
  depends_on "meson" => :build
  depends_on "ninja" => :build
  depends_on "gobject-introspection"
  depends_on "graphite2"
  depends_on "icu4c"
```